src/main/resources/openapiv2.json

```diff
@@ -0,0 +1,889 @@
+                                            "leader": false,
+                                            "in_sync": true
+                                        },
+                                        {
+                                            "broker": 2,
+                                            "leader": true,
+                                            "in_sync": true
+                                        },
+                                        {
+                                            "broker": 3,
+                                            "leader": false,
+                                            "in_sync": false
+                                        }
+                                    ]
+                                }
+                            ]
+                        }
+                    },
+                    "404": {
+                        "description": "The specified topic partition was not found."
+                    },
+                    "422": {
+                        "description": "Unprocessable Entity"
+                    }
+                }
+            },
+            "parameters": [
+                {
+                    "name": "topicname",
+                    "in": "path",
+                    "description": "Name of the topic containing the partition to which you want to send records.",
+                    "required": true,
+                    "type": "string"
+                },
+                {
+                    "name": "partitionid",
+                    "in": "path",
+                    "description": "ID of the partition to which you want to send records.",
+                    "required": true,
+                    "type": "string"
+                }
+            ]
+        },
+        "/consumers/{groupid}/instances/{name}/positions": {
+            "post": {
+                "tags": [
+                    "Consumers"
+                ],
+                "description": "Configures a subscribed consumer to fetch offsets from a particular offset ID the next time it fetches a set of records from a given topic partition. This overrides the default fetch behavior for consumers. You can specify one or more topic partitions.",
+                "operationId": "seek",
+                "consumes": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "body",
+                        "in": "body",
+                        "description": "List of partition offsets from which the subscribed consumer will next fetch records.",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/Offsets"
+                        }
+                    }
+                ],
+                "responses": {
+                    "204": {
+                        "description": "No content."
```

Comment:
"Seek done successfully" ? @laidan6000 @PaulRMellor

Reply:
First offset found successfully?

Reply:
It's not about searching offset but it's about seeking to the offset.

Reply:
Then 'Seek performed successfully'?